packages/core/src/__tests__/ceramic-query-response.test.ts

```diff
@@ -74,14 +74,14 @@ describe('Response to pubsub queries handling', () => {
 
     // Make sure we have some random valid CIDs to respond to pubsub queries with
     for (let i = 0; i < 10; i++) {
-      const commit = await TileDocument.makeGenesis(ceramic, { foo: i }, null)
+      const commit = await TileDocument.makeGenesis(ceramic.signer(), { foo: i }, null)
```

Comment:
this change shouldn't be necessary and indicates that we've made a breaking change.  I believe it's because `TileDocument.makeGenesis` takes the `CeramicSigner` as it's first arg, but this PR redefines what `CeramicSigner` is.  `CeramicSigner` used to extend `CeramicApi` and so passing a `Ceramic` instance was fine.  Now `CeramicSigner` is more a more specific type.  I think we need to update `TileDocument.makeGenesis` (and any other public client-facing functions that used to take the old `CeramicSigner` interface) to take a `CeramicWriter` instead.

Reply:
I think this same issue likely exists with all the other Stream clients, not just TileDocument, it's just most obvious with TileDocument because that's what our tests use the most.